.github/workflows/master.yml

```diff
@@ -0,0 +1,19 @@
+name: Master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  docker-push:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build
+        run: docker build -t eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest .
+      - name: Push
+        run: |
+          echo "${{ secrets.DOCKER_GCR_PASSWORD }} | base64 --decode | docker login -u ${{ secrets.DOCKER_GCR_USERNAME }} --password-stdin https://eu.gcr.io
+          echo "Pushing with tag [latest]"
+          docker push eu.gcr.io/census-eq-ci/eq-survey-runner-benchmark:latest
```

Comment:
Only comment would be is whether `eu.gcr.io/census-eq-ci` should be added as a:
`"${{ secrets.DOCKER_GCR_REGISTRY }}"`

@JamesGardiner Thoughts?

Reply:
I think we should be consistent with other repos and push to Dockerhub rather than GCR.

Reply:
Done!👍